Klicky for Legacy

Maverick-3D opened the pull request:
Modified Klicky-Probe to use on a Voron Legacy.

jlas1 reviewed (changes requested):
Please remove the Zip from the main step file, it should be in the usermods directory.
Can you also remove the STL files that are already in the main repository, it duplicates and can confuse.
Does the Legacy_Dock_Mount_Variable_pt2.stl print well, it has a big overhang?

Maverick-3D commented:
> Please remove the Zip from the main step file, it should be in the usermods directory. 
**Oops! Did not mean to do that. It's removed now.**

> Can you also remove the STL files that are already in the main repository, it duplicates and can confuse. 
**Certainly!**

> Does the Legacy_Dock_Mount_Variable_pt2.stl print well, it has a big overhang?
**It did for me, however, I modified the part and added it to make it easier for others.** 

**Question - do I need to push the changes or will it update the repo automatically?**

jlas1 commented:
> **Question - do I need to push the changes or will it update the repo automatically?**

No, it's automatically updated.
Looks good, i'll merge it
